src/odemis/acq/stream/_sync.py

```diff
@@ -1600,6 +1802,13 @@ def _runAcquisitionScanStage(self, future):
                             leech_np[li] = np
                             if self._acq_state == CANCELLED:
                                 raise CancelledError()
+                    #TODO K.K. just added code here for scan stage, check if placed right
```

Comment:
That looks alright. => remove this comment.

Reply:
oeps that was actually an comment for my-future-selfto review my-past-self